travis/.rubocop_rspec_base.yml

```diff
@@ -114,10 +114,61 @@ SpaceAroundEqualsInParameterDefault:
 StringLiterals:
   Enabled: false
 
+# By convention, RSpec examples use expect { ... } rather than expect do ... end
+Style/Blocks:
+  Exclude:
+    - spec/**/*
+
+# In tests, it is convenient to define a test class like RSpec::Something on a single line
+Style/ClassAndModuleChildren:
+  Exclude:
+    - spec/**/*
+
+# Our tests use compact class definitions to save "real estate" on the scren.
+Style/EmptyLineBetweenDefs:
+  Exclude:
+    - spec/**/*
+Style/EmptyLinesAroundAccessModifier:
+  Exclude:
+    - spec/**/*
+
+Style/GlobalVars:
+  Exclude:
+    - spec/**/*
+    - benchmarks/**/*
+
+# Allow use of: expect { ... }.to raise_error(e) { ... }
+Style/MultilineBlockChain:
+  Exclude:
+    - spec/**/*
+
+# Allow use of: something rescue nil
+Style/RescueModifier:
+  Enabled: false
+
+# Our tests use single-line methods in numerous places. This saves "real estate" on the screen.
+Style/Semicolon:
+  Exclude:
+    - spec/**/*
+Style/SingleLineMethods:
+  Exclude:
+    - spec/**/*
+
+# We name block parameters what we want
+Style/SingleLineBlockParams:
+  Enabled: false
+
+# RSpec uses {:a => 1} rather than { :a => 1 }
```

Comment:
I think David tended to use `{:a => 1}` but I've always used `{ :a => 1 }` so our code is a bit of a frankenstein here.